crd-docs/crd/kiali.io_kialis.yaml

```diff
@@ -1126,6 +1126,9 @@ spec:
                   cache_token_namespace_duration:
                     description: "This Kiali cache is a list of namespaces per user. This is typically a short-lived cache compared with the duration of the namespace cache defined by the `cache_duration` setting. This is specified in seconds."
                     type: integer
+                  token_expire_duration:
+                    description: "This is the time, in seconds, that the service account token will be refreshed"
+                    type: integer
```

Comment:
trivial - sort this, by moving below qps setting

Reply:
I wont' add this comment everywhere else - but you get the idea... this is trivial, but just make the settings sorted alphabetically.

Reply:
Update this and the other changed files